docs/README.md

```diff
@@ -5,7 +5,7 @@
 This directory contains the following documents that relate to the project:
 
 - configuration:
-  - [Theme Modification](./configuration/theme-modification.md) describes how to edit a theme of the whole AsyncAPI component or of its specific part.
+  - [Style Modification](./configuration/style-modification.md) describes how to edit the styles of the AsyncApi component.
```

Comment:
```suggestion
  - [Style Modification](./configuration/style-modification.md) describes how to edit the styles of the AsyncAPI component.
```

Reply:
here and everywhere I guess, I noticed now that it is written in lowercase in some places